AutomatedTesting/Gem/PythonTests/prefab/Prefab_BasicWorkflow_CreateAndDeletePrefab.py

```diff
@@ -0,0 +1,36 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def Test():
```

Comment:
This must be called the same as the file. In this case: Prefab_BasicWorkflow_CreateAndDeletePrefab

Reply:
I think you only need to have the function name that's the same as one that's used at the last line of the test file (`Report.start_test(<test_name>)`).

Reply:
Will fix it for debug logs.

Reply:
Fixed.